### DISCUSSION TITLE
Ignore Maintenance Windows for deploy State or Action Chain

### DISCUSSION THREAD

**Discussion by vega82** [uyuni-project/uyuni#6486] [Q&A] [answered]
Hi,

is there any way to deploy highstate or run an action chain outside a maintenance window?
I still looking for a way to push new created channels timely to some clients.

My created maintenance windows are created to schedule system updates only for a specific time range.
But for deploying some new configs, etc. i allways need to wait for arriving this Maintenance Window or kick the systems from Maintenance Windows, deploy my settings and add the system again to Maintenance Window.

I didn't found another solution in documentation, to do this more effective.
So looking if somebody can give me a hint.

**Comment by mcalmer** [comment, marked as answer]
You want to do changes outside of a maintenance window. So why do you want the maintenance window at all if you want to ignore it at any point in time?

This is a bit against the idea of a maintenance window, right?

As you can do in a state.apply everything, it does not make sense to remove it out of the maintenance window. It would be the same as remove the client from the maintenance window.
So, no. Not possible.

**Comment by vega82** [comment]
So maybe i have a missunderstanding. I have maintenance windows to be sure, updates where autodeployed only within this time.
This ist not possible by any other way, i think ?

But sometime i'll rollout new features and setting to the whole infrastructure or groups of them. And this should be done right off independently from a maintenance windows.

So there should be a way to trigger auto OS-updates by another schedule or have an option to skip maintenance windows for a onetime action.

I have about 100 systems in uyuni and it's no option to put everty system out of his maintenance window for such a deployment.

**Comment by mcalmer** [comment]
Maintenance Window mean: changes to the system only in this timeframe. Outside of a maintenance window no change should be done to the system. No package change, no config file change which affects running services nor restart of services.

You can always qickly define a new mainetance window. Just change the calendar and introduce a new maintenance window.

**Reply by vega82** [reply]
Ok i'll do it like this.
But i think it makes sense to create a feature request for this case.
So a checkbox with "run immediately and ignore maintenance window" would be usefull in some cases.
